### DISCUSSION TITLE
Docs API accordions should be expanded when there are > 0 APIs

### DISCUSSION THREAD

**Discussion by bennypowers** [RedHat-UX/red-hat-design-system#1605] [Documentation feedback]
Whenever a docs page has an API accordion, every accordion panel which has a positive number of members should come expanded

![image](https://github.com/RedHat-UX/red-hat-design-system/assets/1466420/42cccb8f-ee22-499e-be81-8d026671b60d)

In this case, All of the accordion panels should come expanded except for "CSS Shadow Parts" and "CSS Custom Properties". We may add an "expand / collapse" all button above or below the accordion.

**Comment by bennypowers** [comment]
cc @zeroedin who did the work to bring this delightful pattern to the api pages
cc @marionnegp for ux guidance

see also #1604

**Comment by marionnegp** [comment]
@bennypowers, what's the thinking behind expanding those accordions by default? Feels like it would be defeat the purpose of putting content in accordions. 

An expand all / collapse all button would be great though. I think that's a pattern that they're exploring for product pages.

**Reply by bennypowers** [reply]
- find in page (aka ctrl-f)
- linking to attributes

**Comment by zeroedin** [comment]
> An expand all / collapse all button would be great though.

I like this idea as well, maybe it is best of both worlds.  

I can see positives and negatives of having expanded as default here.  At some rate depending how you are searching the page you may `ctrl + f` and because that content is internal to a closed accordion the quick find doesn't find the term.  The other flip side is the cognitive load IMHO when understand your position within the page and what docs are you looking at when multiple sub components are on the page.  

I would ultimately like to get TOC working on the page giving quick jump feature set which might mitigate that second issue.
